dojo/tools/chefinspect/parser.py

```diff
@@ -0,0 +1,58 @@
+import json
+from dojo.models import Finding
+
+
+class ChefInspectParser(object):
+    def get_scan_types(self):
+        return ["Chef Inspect Log"]
+
+    def get_label_for_scan_types(self, scan_type):
+        return scan_type
+
+    def get_description_for_scan_types(self, scan_type):
+        return """Chef Inspect log file"""
+
+    def convert_score(self, raw_value):
+        val = float(raw_value)
+        if val == 0.0:
+            return "Info"
+        elif val < 4.0:
+            return "Low"
+        elif val < 7.0:
+            return "Medium"
+        elif val < 9.0:
+            return "High"
+        else:
+            return "Critical"
```

Comment:
@manuel-sommer my apologies. I approved, and then dismissed it after seeing the feedback from @cneill was not implemented. Reasoning for the dismissed approval is in this comment
https://github.com/DefectDojo/django-DefectDojo/pull/9329#discussion_r1456069356

Reply:
Whoops, good catch!